internal/provider/provider.go

```diff
@@ -234,25 +251,30 @@ func (p *IncusProvider) Configure(ctx context.Context, req provider.ConfigureReq
 	// in Terraform configurations where the Incus remote might not
 	// exist yet.
 	for _, remote := range data.Remotes {
-		port := remote.Port.ValueString()
-		if port == "" {
-			port = "8443"
+		isDefault := false
+
+		protocol := remote.Protocol.ValueString()
+		if protocol == "" {
+			protocol = "incus"
 		}
 
-		scheme := remote.Scheme.ValueString()
-		if scheme == "" {
-			scheme = "unix"
+		auth_type := remote.Auth_Type.ValueString()
+		if auth_type == "" {
+			auth_type = "tls"
 		}
 
 		incusProviderRemoteConfig := provider_config.IncusProviderRemoteConfig{
-			Name:    remote.Name.ValueString(),
-			Token:   remote.Token.ValueString(),
-			Address: remote.Address.ValueString(),
-			Port:    port,
-			Scheme:  scheme,
+			Name:      remote.Name.ValueString(),
+			Address:   remote.Address.ValueString(),
+			Protocol:  protocol,
+			Auth_Type: auth_type,
```

Comment:
Please follow our naming convention (style and use complete words): `AuthenticationType` and `authenticationType`